src/Lecture1.hs

```diff
@@ -123,4 +149,8 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> String
+lowerAndGreater n list = show n ++ " is greater than " ++ show lower ++ " elements and lower than " ++ show greater ++ " elements"
+    where
+        greater = length $ filter ( > n) list
+        lower = length $ filter ( < n) list
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
You can write it more performant with only one traversal of the list 🙂
It's desirable to write functions that traverse lists only a single time and consume elements only once. In that case, the composition of such functions will have nice streaming and lazy properties.

Reply:
Thanks! Will think of one that traverses the list once.